spec/models/video_spec.rb

```diff
@@ -1,43 +1,35 @@
 require 'spec_helper'
 
 describe Video do
+  it { should belong_to(:category) }
+  it { should validate_presence_of(:title) }
+  it { should validate_presence_of(:description) }
+
+  let(:first_video) { Fabricate(:video, title: "The first title", description: "The first description", created_at: 1.day.ago) }
+  let(:second_video) { Fabricate(:video, title: "The second title", description: "The second description") }
+
   it "saves itself" do
```

Comment:
This is not in this PR, but I would delete it - it tests ActiveRecord#save a code that you have not written.